sylvia/tests/ui/attributes/messages/invalid_custom.rs

```diff
@@ -0,0 +1,33 @@
+#![allow(unused_imports)]
+use sylvia::cw_std::{Response, StdResult};
+use sylvia::types::InstantiateCtx;
+
+mod interface {
+    use sylvia::cw_std::StdError;
+
+    #[sylvia::interface]
+    pub trait Interface {
+        type Error;
+    }
+
+    impl Interface for crate::Contract {
```

Comment:
nit: Let's move the interface impl to the contract's module

Reply:
This is simplified as this is not supposed to be an example.